api-reference/10 UI Components/dxSelectBox/1 Configuration/customItemCreateEvent.md

```diff
@@ -8,234 +8,8 @@ default: 'change'
 Specifies the <a href="https://en.wikipedia.org/wiki/DOM_events" target="_blank">DOM event</a> after which the custom item should be created. Applies only if [acceptCustomValue](/api-reference/10%20UI%20Components/dxSelectBox/1%20Configuration/acceptCustomValue.md '{basewidgetpath}/Configuration/#acceptCustomValue') is enabled.
 
 ---
-The default custom item creation event is `change`, and items are created when users press the **Enter** key or when the component loses focus.
+This property accepts a single event name or several names separated by spaces.
 
-To allow users to create items only when they press the **Enter** key, pass an empty string to this property.
+The recommended events are "keyup", "blur", "change", "input", and "focusout", but you can use other events as well. The default value of this property is "change", and items are created when the component loses focus. 
```

Comment:
Let's also add an information for users who want to create custom events on `tab` press. In this case they can specify `focusout` or `change` as a `customItemCreateEvent`